From 933fdfb35c23685ed2b578733287a19ff5d2cb7d Mon Sep 17 00:00:00 2001
From: Sarun Intaralawan <sarunint@sarunint.com>
Date: Thu, 23 Oct 2025 15:36:09 +0700
Subject: [PATCH 1/2] fix: import CorePrivate

---
 src/framework/accessibility/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/framework/accessibility/CMakeLists.txt b/src/framework/accessibility/CMakeLists.txt
index cdb8c08923..38c830ef48 100644
--- a/src/framework/accessibility/CMakeLists.txt
+++ b/src/framework/accessibility/CMakeLists.txt
@@ -48,6 +48,8 @@ set(MODULE_SRC
 
 if (OS_IS_LIN)
     # it needed to send a spontaneous event by activating navigation
+    find_package(Qt6 REQUIRED COMPONENTS CorePrivate)
+    set(MODULE_LINK ${MODULE_LINK} Qt6::CorePrivate)
     set(MODULE_INCLUDE_PRIVATE ${Qt6Core_PRIVATE_INCLUDE_DIRS})
 endif()
 
-- 
2.51.0

